emit_translations.py

```diff
@@ -82,16 +83,25 @@ def main():
                     help='Output directory for translated source files.')
     ap.add_argument('-v', '--verbose', action='store_true',
                     help='Enable verbose logging')
+
+    # Translation args
+    ap.add_argument('--int-size', type=int, choices=list(IntSize), default=IntSize.Int32,
```

Comment:
I like the idea of using the `IntSize` enum to provide the list of choices to the user, but according to the [docs on `argpase`](https://docs.python.org/3/library/argparse.html#choices) it's not recommended to use `enum` types to directly construct lists of choices because it makes it's hard to control the appearance of the options (run `python3 emit_translation.py --help` to see what I mean).
I think you could acheive the desired effect with the following:

```suggestion
    ap.add_argument('--int-size', type=int, choices=[size.value for size in IntSize], default=IntSize.Int32,
```

Reply:
Interesting about the help message. On my system Python version (3.12) I had help display properly (displayed {16, 32} as the choices) so I didn't notice anything. But on 3.10 I did get the full enum name.

Seems like that may be because [`__str__()`  was changed to use `int.__str__()` for IntEnum's](https://docs.python.org/3/library/enum.html#enum.IntEnum) in 3.11. Good catch though, and more reason for me to test on 3.10 specifically